app/src/components/market/common_sections/tables/outcome_table/index.tsx

```diff
@@ -124,6 +124,7 @@ export const OutcomeTable = (props: Props) => {
 
   const TableCellsAlign = ['left', 'left', 'right', 'right', 'right', 'right', 'right']
   const symbol = useSymbol(collateral)
+  const bondCollateral = relay ? 'DAI' : 'ETH'
```

Comment:
@Frankaus Because of the way you made it..This test market on xDai is showing eth
http://localhost:3001/#/0xebb1b67bafbb74a22e293b636ee66d2cbc865f9f/finalize
![Screenshot 2021-08-18 at 17 31 07](https://user-images.githubusercontent.com/33226956/129927003-77ac93e5-e887-4eca-92db-705797b6472c.png)
So the simplest solution is to use nativeAsset variable which is already present in this file...